### PR TITLE
[nixos-20.03] pingus: 0.7.6 -> unstable

### PR DESCRIPTION
**Pull request by woffs**
###### Motivation for this change
Backport #83706 to fix pingus in nixos-20.03

###### Things done

- [X] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [X] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [X] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by aanderse** [comment]
So does `pingus` have music working again? See https://github.com/NixOS/nixpkgs/pull/80817.

File: pkgs/games/pingus/default.nix
Comment (aanderse) [inline pkgs/games/pingus/default.nix:8]: `cmake` should be in `nativeBuildInputs`.

**Review by aanderse — CHANGES_REQUESTED** [review]

File: pkgs/games/pingus/default.nix
Comment (woffs) [reply pkgs/games/pingus/default.nix:8]: this should be fixed in master, as this is only a backport

**Comment by woffs** [comment]
> So does `pingus` have music working again? See #80817.

music works. `pingus` did not even compile before (`boost` related failure)

File: pkgs/games/pingus/default.nix
Comment (aanderse) [reply pkgs/games/pingus/default.nix:8]: Agreed, so maybe this PR should wait until a new PR against `master` is merged.

Comment (woffs) [reply pkgs/games/pingus/default.nix]: I've submitted one and cherry-picked it here.

**Pull request merged**
